src/locales/zh_CN/filter-bar.ts

```diff
@@ -0,0 +1,21 @@
+import { SimpleTranslationEntries } from "#app/interfaces/locales";
+
+export const filterBar: SimpleTranslationEntries = {
+  "genFilter": "世代",
+  "typeFilter": "属性",
+  "unlocksFilter": "解锁",
+  "winFilter": "通关",
+  "sortFilter": "排序",
+  "all": "全部",
+  "normal": "通常",
```

Comment:
```suggestion
  "normal": "无闪光",
```
 a feedback from cn community suggesting that normal should be like "not shiny", more understandable

Reply:
thanks! cn cummunity!